src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -79,10 +79,13 @@ public SnowflakeSinkConnector() {
    */
   @Override
   public void start(final Map<String, String> parsedConfig) {
+    // ensure we start counting tasks at 0 for this instance
+    SnowflakeSinkTask.setTotalTaskCreationCount(0);
+
     Utils.checkConnectorVersion();
 
-    // initialize logging with correlationId
-    LoggerHandler.setCorrelationUuid(UUID.randomUUID());
+    // initialize logging with global instance Id
+    LoggerHandler.setConnectGlobalInstanceId(UUID.randomUUID());
```

Comment:
Do we really need the full UUID? it seems to long and makes the log even harder to read

Reply:
@sfc-gh-japatel  and I discussed this before, we'd definitely prefer a way to guarantee uniqueness with a shorter UUID, but we didn't find a great way to achieve that. Open to ideas though!

Reply:
Given the number of instance being small, i think we will never get a collision if we just take the last section, this is like the chance of a hash collision....Or we could take something from the current timestamp to even lower the chance...

Reply:
I changed it to hashcode on a UUID and the current time in millis